Sources/ParselyTracker.swift

```diff
@@ -14,6 +14,7 @@ public class Parsely {
             return nil
         }
     }
+    // FIXME: Is it appropriate to have `sharedInstance` and `getInstance` (below) both instantiating a new `Parsely`?
```

Comment:
I believe the `getInstance` is meant for unit tests, so that each test has their own test instances.